pandas/core/window/rolling.py

```diff
@@ -1542,12 +1579,12 @@ def median(self, **kwargs):
         window_func = self._get_roll_func("roll_median_c")
         # GH 32865. Move max window size calculation to
         # the median function implementation
-        return self._apply(window_func, center=self.center, name="median", **kwargs)
+        return self._apply(window_func, center=False, name="median", **kwargs)
```

Comment:
This needed to be changed to `False`?

Reply:
Good catch, no this was from an old attempt at a fix